plugin.php

```diff
@@ -31,9 +31,10 @@
     $cron_excecuted = [];
     $scheme         = defined( 'REQUEST_SCHEME' ) ? REQUEST_SCHEME : 'https';
 
+    // Multisite
     if ( defined( 'WP_ALLOW_MULTISITE' ) && WP_ALLOW_MULTISITE === true ) {
         global $wpdb;
-        $sql = $wpdb->prepare( "SELECT domain FROM $wpdb->blogs WHERE archived='0' AND deleted ='0' LIMIT 0,300", '' );
+        $sql = "SELECT domain, path FROM $wpdb->blogs WHERE archived=0 AND deleted=0";
```

Comment:
Why is the prepare statement removed?

Reply:
It was used incorrectly without placeholders and returned error

Reply:
Well that's funky. Maybe it is just better to leave it out here. 👍